AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -301,7 +302,10 @@ extension FormValueItem where ValueType == String {
 
 extension CardViewController: CardViewControllerProtocol {
     func update(storePaymentMethodFieldVisibility isVisible: Bool) {
-        items.storeDetailsItem.value = false
```

Comment:
Thanks again for your PR @eduardourso !
It would be logical to keep this line. If the switch is hidden, we should not store the info for next time.

Reply:
sure, that makes sense. I'll revert this change.

Reply:
@erenbesel I've updated it here, to consider the visibility: [a536d8c](https://github.com/Adyen/adyen-ios/pull/875/commits/a536d8cfb27152031e6c4a52287abf2b63795dfd)